File: python-client-sdk/src/sdk/models/shared/invalidparam.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class InvalidParam:
    r"""InvalidParam
    Information about an input parameter pass to the API call that could not be parsed.
    """
    
    data_type: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('dataType') }})
    input_cell_type: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('inputCellType') }})
    param_name: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('paramName') }})
    param_value: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('paramValue') }})
    

File: java-client-sdk/lib/src/main/java/org/openapis/openapi/models/shared/InvalidParam.java
package org.openapis.openapi.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * InvalidParam
 * Information about an input parameter pass to the API call that could not be parsed.
**/
public class InvalidParam {
    @JsonProperty("dataType")
    public String dataType;
    public InvalidParam withDataType(String dataType) {
        this.dataType = dataType;
        return this;
    }
    
    @JsonProperty("inputCellType")
    public String inputCellType;
    public InvalidParam withInputCellType(String inputCellType) {
        this.inputCellType = inputCellType;
        return this;
    }
    
    @JsonProperty("paramName")
    public String paramName;
    public InvalidParam withParamName(String paramName) {
        this.paramName = paramName;
        return this;
    }
    
    @JsonProperty("paramValue")
    public String paramValue;
    public InvalidParam withParamValue(String paramValue) {
        this.paramValue = paramValue;
        return this;
    }
    
}
